Makefile

```diff
@@ -83,31 +83,32 @@ base:
 		$(COMMON_ARGS)
 	@docker load < build/$@.tar
 
-kernel:
+kernel: buildkitd
 	@buildctl --addr $(BUILDKIT_HOST) \
 		build \
 		--exporter=local \
 		--exporter-opt output=build \
 		--frontend-opt target=$@ \
 		$(COMMON_ARGS)
 
-initramfs:
+initramfs: buildkitd
 	@buildctl --addr $(BUILDKIT_HOST) \
 		build \
 		--exporter=local \
 		--exporter-opt output=build \
 		--frontend-opt target=$@ \
 		$(COMMON_ARGS)
 
-rootfs: hyperkube etcd coredns pause osd trustd proxyd blockd
+rootfs: buildkitd hyperkube etcd coredns pause osd trustd proxyd blockd
 	@buildctl --addr $(BUILDKIT_HOST) \
 		build \
 		--exporter=local \
 		--exporter-opt output=build \
 		--frontend-opt target=$@ \
 		$(COMMON_ARGS)
 
-installer:
+installer: buildkitd
+	@mkdir build
```

Comment:
This is added because trying to `make installer` on a new clone or after a `make clean` fails with 
```
error: invalid exporter-opt: output: open build/installer.tar: no such file or directory
make: *** [Makefile:111: installer] Error 1
```